api-schema/src/main/java/org/stellar/anchor/api/rpc/method/NotifyCustomerInfoUpdatedRequest.java

```diff
@@ -9,4 +9,6 @@
 @SuperBuilder
 @AllArgsConstructor
 @EqualsAndHashCode(callSuper = false)
-public class NotifyCustomerInfoUpdatedRequest extends RpcMethodParamsRequest {}
+public class NotifyCustomerInfoUpdatedRequest extends RpcMethodParamsRequest {
+  String status;
+}
```

Comment:
Should SEP-31 transactions also follow the same logic? Currently, calling this RPC always changes the transaction status to `PENDING_RECEIVER`. Adding a status field would allow us to preserve the original behavior.

Anyway, I believe the answer is yes. We want SEP-31 to call `GET /customer` and update the transaction status based on the response.

Reply:
Yes I think thats correct.